go/pkg/filemetadata/filemetadata_test.go

```diff
@@ -306,6 +300,44 @@ func TestComputeSymlinksToDirectory(t *testing.T) {
 	}
 }
 
+func createFileWithXattr(t *testing.T, fileContent, xattrName, xattrValue string) (string, error) {
```

Comment:
Since this is a helper function, let's use `t.Helper()` and `t.Fatal` instead of returning an error.